src/openms/source/ANALYSIS/ID/SimpleSearchEngineAlgorithm.cpp

```diff
@@ -462,35 +454,28 @@ void SimpleSearchEngineAlgorithm::postProcessHits_(const PeakMap& exp,
         if (!peptide_motif_.empty() && !boost::regex_match(current_peptide, peptide_motif_regex)) { continue; }          
       
         bool already_processed = false;
-#ifdef _OPENMP
-#pragma omp critical (processed_peptides_access)
-#endif
+        #pragma omp critical (processed_peptides_access)
         {
           // peptide (and all modified variants) already processed so skip it
           if (processed_petides.find(c) != processed_petides.end())
           {
             already_processed = true;
           }
-	  else
-	  {
+          else
+          {
             processed_petides.insert(c);
-	  }
+          }
         }
 
         // skip peptides that have already been processed
         if (already_processed) { continue; }
 
-#ifdef _OPENMP
-#pragma omp atomic
-#endif
         ++count_peptides;
```

Comment:
for sanity maybe we can annote that this is atomic?

```suggestion
        ++count_peptides; // atomic update
```